core/controllers/base.py

```diff
@@ -367,7 +367,8 @@ def _render_exception_json_or_html(self, return_type, values):
                     'error-iframed.mainpage.html',
```

Comment:
Should we have also multiple versions of iframed page?

Reply:
No.
The iframed page is only used when an exploration with wrong exploration ID is loaded through an iframe.
And does not load error-page.directive.html which needs status code.

Reply:
Okay